### PR TITLE
Create hangouts view 140313371

### LINKED ISSUE #17: Hangouts View
### Scenario

As a user I should be able to view a list all hangouts and my hangout

### LINKED ISSUE #19: Hangout Shuffle
### Scenario

As an admin user I should be able to shuffle hangouts.

### PR DESCRIPTION
**Pull request by wanjiru-w**
#### What does this PR do?
It adds a table to the hangouts page and uses mock data to display Andelans names on the page.

#### How should this be manually tested?
By clicking on the hangouts link on the menu.

Closes #17

### CODE REVIEW

**Review by akabiru — APPROVED** [review]
LGTM 👍

File: src/Common/Nav.elm [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module Common.Nav exposing (navBar)
```
Comment (waiyaki) [inline src/Common/Nav.elm:1]: @andela-pwanjiru @akabiru Any particular reason we are reintroducing this file? 🤔 
I think this functionality is taken care of in the [app common nav](https://github.com/AndelaOSP/shufflebox-frontend/blob/develop/src/App/Common/Nav.elm).

Comment (akabiru) [reply src/Common/Nav.elm:1]: @waiyaki Bruh, you need to sleep. 😄  I think there are two "NavBars", one for the landing page and one for the "app" page.

Comment (waiyaki) [reply src/Common/Nav.elm:1]: @akabiru Yeah, there are. The navbar for the landing page is in `src/Landing/Nav.elm` while that of the app page is in `src/App/Common/Nav.elm`. I think we'd moved the file in question from `src/Common/` to `src/App/Common` when we refactored and broke the app into subviews. Unless I'm mistaken 😄. In which case I really need to go sleep. 😴

**Pull request merged**